our_site/planner/views.py

```diff
@@ -1,33 +1,69 @@
 from django.http import HttpResponse, HttpResponseRedirect
 from django.template import loader
+from .models import Event
 import django.forms as forms
+from django.shortcuts import render
 
 
-class DashboardFilterAllEvents(forms.Form):
-    pass
 
+def dashboard(request): 
+    owner = None
+    event_list = []
 
-class DashboardFilterMyEvents(forms.Form):
-    pass
+    class DashboardFilterAllEvents(forms.Form):
```

Comment:
Why is this class (as well as the others) defined inside the view?

Reply:
that is just where the classes were when i merged from main, so i kept them there